pkg/config/config.go

```diff
@@ -416,6 +416,7 @@ type Ko struct {
 	CreationTime        string            `yaml:"creation_time,omitempty" json:"creation_time,omitempty"`
 	KoDataCreationTime  string            `yaml:"ko_data_creation_time,omitempty" json:"ko_data_creation_time,omitempty"`
 	SBOM                string            `yaml:"sbom,omitempty" json:"sbom,omitempty"`
+	SBOMDir             string            `yaml:"sbom_dir,omitempty" json:"sbom_dir,omitempty"`
```

Comment:
Maybe better to call this `SBOMDirectory` and/or the property `sbom_directory`? 🤔

Reply:
yes, no problem.